src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -583,4 +581,14 @@ std::string LiveFunctionsDataView::GetToolTip(int row, int column) {
   return all_scope_ids;
 }
 
+void LiveFunctionsDataView::SetScopeStatsCollection(
+    std::shared_ptr<orbit_client_data::ScopeStatsCollectionInterface> scope_stats_collection) {
```

Comment:
Is there a reason for this being a shared_ptr? Wouldn't a unique_ptr be enough here?